appveyor.approval.yml

```diff
@@ -0,0 +1,18 @@
+version: '{build}'
+
+environment:
+    PATH: 'C:\Ruby200\bin;%PATH%'
+
+install:
+    - cmd: bundle install
+
+build_script:
+    - cmd: build.netstd.cmd approve

```

Comment:
Yes, I think we'll have to do that anyway. We can update CONTRIBUTING.md to explain the approval process.